Seminar2/Task2.py

```diff
@@ -0,0 +1,23 @@
+#Напишите программу, которая получает целое число и возвращает его 
+#шестнадцатеричное строковое представление. Функцию hex используйте для проверки своего результата.
+
+num = 189
+def func_hex(num):
+    result = []
+    letters = ['a', 'b', 'c', 'd', 'i', 'f']
+    
+    while num > 0:
+        x = num % 16
+        
+        if x >= 10:
+            x = letters[int(x-10)] 
+            result.append(str(x))
+        else:
+            result.append(str(x))
```

Comment:
Внимательно изучите предложенный мной код
```suggestion
    while num:
        result.append(letters[num % 16])
```